packages/aragon-api-react/README.md

```diff
@@ -0,0 +1,173 @@
+# aragonAPI for React
+
+This module lets you interact with aragonAPI using a [React Hook](https://reactjs.org/docs/hooks-intro.html) or, alternatively, with a [render prop](https://reactjs.org/docs/render-props.html).
+
+[`@aragon/api`](https://github.com/aragon/aragon.js/blob/master/docs/APP.md) is used under the hood, so being familiar with it can be useful.
+
+## Usage
+
+```jsx
+import { ConnectAragonApi, useAragonApi } from '@aragon/api-react'
+
+function App() {
+  const { api, appState } = useAragonApi()
+  const { count = 0 } = appState
+
+  return (
+    <div>
+      <div>{count}</div>
+      <button onClick={() => api.increment(1)}>
+        Increment
+      </button>
+    </div>
+  )
+}
+
+ReactDOM.render(
+  <ConnectAragonApi>
+    <App />
+  </ConnectAragonApi>,
+  document.getElementById('root')
+)
+
+```
+
+## `<ConnectAragonApi />`
+
+Before using any Hook provided, you need to declare this component to connect the app. It is generally a good idea to do it near the top level of your React tree. It should only be declared once.
+
+It has an optional `reducer` prop, which lets you process the state coming from the background script. If not provided, the state is passed as is from the background script.
+
+### Example
+
+```jsx
+import BN from 'bn.js'
+import { ConnectAragonApi, useAragonApi } from  '@aragon/api-react'
+
+function App() {
+  const { appState } = useAragonApi(reducer)
+  return (
+    <div>{appState.balance.toString(10)}</div>
+  )
+}
+
+ReactDOM.render(
+  <ConnectAragonApi
+    reducer={state => {
```

Comment:
What do you think about declaring the reducer function outside the component?

This is based on the documentation patterns I see in https://redux.js.org/advanced/example-reddit-api

Reply:
Yes let’s do this 👍 I was trying to make it as compact as possible for the documentation, but it looks better outside of it.